RemoteIDModule/status.cpp

```diff
@@ -215,16 +221,16 @@ String status_json(void)
         { "SYSTEM:AreaFloor", String(UAS_data.System.AreaFloor) },
```

Comment:
It would probably make sense to then also fix the SYSTEM:AreaCeiling, SYSTEM:AreaFloor and LOCATION:Height at the same time?

Reply:
I agree. I will also change the Area Ceiling and Area Floor.